src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements FigureArea {
+    private final int height;
+    private final int width;
+
+    public Rectangle(int height, int width, String color, String property) {
+        setColor(color);
+        setProperty(property);
+        this.height = height;
+        this.width = width;
+        setPropertyLength(width);
+        areaCalculate();
+    }
+
+    @Override
+    public void areaCalculate() {
+        setArea(height * width);
+    }
```

Comment:
```suggestion
    public double calculateArea() {
        return height * width;
    }
```

same everywhere in child figure classes